registry/handlers/app.go

```diff
@@ -470,7 +471,9 @@ func (app *App) configureEvents(configuration *configuration.Configuration) {
 	// replacing broadcaster with a rabbitmq implementation. It's recommended
 	// that the registry instances also act as the workers to keep deployment
 	// simple.
-	app.events.sink = notifications.NewBroadcaster(sinks...)
+	// NOTE(jmwong): Always use SyncBroadcaster here and delegate whether or not the events a processed asynchronously
```

Comment:
@manishtomar 

a -> are -- also consider using your github alias since you're the commiter